src/domain/orders/details/utils/create-filtering.js

```diff
@@ -1,4 +1,4 @@
-export const filterItems = (order, isClaim) => {
+export const filterItems = order => {
```

Comment:
It seems, that I cannot receive the items from the claim here in the admin. Look at the video below. It only allows me to receive a single item.
https://user-images.githubusercontent.com/59018053/135453863-cfd50153-bab6-4408-be19-14c6d75cee81.mov